sgx/demo/enclave/kmyth_sgx_retrieve_key_demo_enclave.edl

```diff
@@ -0,0 +1,6 @@
+enclave
+{
+  from "kmyth_enclave.edl" import *;
+	from "sgx_tsgxssl.edl" import *;
```

Comment:
GitHub is showing me an odd spacing here, not sure if it's real or not.

Reply:
I think I fixed this by editing in GitHub